cloudformation/cw-dashboard-substack.cfn.yaml

```diff
@@ -0,0 +1,241 @@
+            - - '{"type":"text","x":0,"y":48,"width":24,"height":1,"properties":{"markdown":"\n#
+                      EC2 Instance Metrics\n"}}'
+              - !Sub ',{"type":"metric","x":0,"y":49,"width":6,"height":6,"properties":{"view":"timeSeries","stacked":false,"metrics":[["AWS/EC2","CPUUtilization","InstanceId","${MasterInstanceID}"]],"region":"${AWS::Region}"}}'
+              - !If # EFS metrics graphs
+                - CreateEFSSubstack
+                - !Join
+                  - ''
+                  - - ',{"type":"text","x":0,"y":28,"width":24,"height":1,"properties":{"markdown":"\n#
+                      EFS Metrics\n"}},' # beginning comma was hard-coded
+                    - !Sub '{"type":"metric","x":0,"y":29,"width":6,"height":6,"properties":{"view":"timeSeries","stacked":false,"metrics":[["AWS/EFS","BurstCreditBalance","FileSystemId","${EFSFileSystemID}"]],"region":"${AWS::Region}"}}'
+                - ''
+              # EBS metrics graphs
+              - ',{"type":"text","x":0,"y":2,"width":24,"height":1,"properties":{"markdown":"\n# EBS Metrics\n"}}'
+              # EBS first graph with 2 volumes max
+              - !Sub
+                - ',{"type":"metric","x":0,"y":3,"width":6,"height":6,"properties":{"view":"timeSeries","stacked":false,"metrics":[["AWS/EBS","VolumeReadOps","VolumeId","${EBS_Volume1}"],[".","VolumeWriteOps",".","."]'
+                - EBS_Volume1: !Select
+                    - '0'
+                    - !Ref 'EBSVolumesIDs'
+              - !If
+                - UseEBSVol2 # If we have a second EBS volume
+                - !Sub
+                  - ',[".","VolumeReadOps",".","${EBS_Volume2}"],[".","VolumeWriteOps",".","."]'
+                  - EBS_Volume2: !Select
+                      - '1'
+                      - !Ref 'EBSVolumesIDs'
+                - ''
+              - !If
+                - UseEBSVol3 # If we have a third EBS volume
+                - !Sub
+                  - ',[".","VolumeReadOps",".","${EBS_Volume3}"],[".","VolumeWriteOps",".","."]'
+                  - EBS_Volume3: !Select
+                      - '2'
+                      - !Ref 'EBSVolumesIDs'
+                - ''
+              - !If
+                - UseEBSVol4 # If we have a fourth EBS volume
+                - !Sub
+                  - ',[".","VolumeReadOps",".","${EBS_Volume4}"],[".","VolumeWriteOps",".","."]'
+                  - EBS_Volume4: !Select
+                      - '3'
+                      - !Ref 'EBSVolumesIDs'
+                - ''
+              - !If
+                - UseEBSVol5 # If we have a fifth EBS volume
+                - !Sub
+                  - ',[".","VolumeReadOps",".","${EBS_Volume5}"],[".","VolumeWriteOps",".","."]'
+                  - EBS_Volume5: !Select
+                      - '4'
+                      - !Ref 'EBSVolumesIDs'
+                - ''
+                # put those only if the previous cond is true? Better to execute less but
```

Comment:
When adding this kind of comments you could add a FIXME or TODO keyword in this way IntelliJ will alert you when committing a file.

Reply:
Done